src/engine/strat_engine/engine.rs

```diff
@@ -35,6 +55,14 @@ impl StratEngine {
     /// Returns an error if there was an error reading device nodes.
     /// Returns an error if there was an error setting up any of the pools.
     pub fn initialize() -> EngineResult<StratEngine> {
+        let minor_dm_version = DM::new()?.version()?.1;
```

Comment:
We might want to document the reason for this requirement in some docs.

Reply:
opened https://github.com/stratis-storage/stratis-docs/issues/72